job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml

```diff
@@ -64,18 +64,17 @@
                   git clean -fdx
                   git reset --hard origin/master
 
-                  make virtualenv
-                  ./venv/bin/pip install --upgrade pip
-                  ./venv/bin/pip install wheel  # this allows pip to cache compiled packages instead of rebuilding them all on every invocation
-                  make requirements
+                  pip install -U digitalmarketplace-developer-tools
+                  invoke virtualenv
+                  invoke requirements
                 ''')
               }
             }
             stage('Generate') {
               dir('digitalmarketplace-scripts') {
                 sh('''
                   [ "$DRY_RUN" = "true" ] && FLAGS="$FLAGS --dry-run"
-                  
+
```

Comment:
very tiny but is this whitespace change intended?

Reply:
Yes, my editor was complaining about it.